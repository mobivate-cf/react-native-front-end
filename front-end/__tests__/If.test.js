import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';

import If from '../src/component/If/If';
Enzyme.configure({ adapter: new Adapter() });

describe('If Component', () => {
  describe('Functionality', () => {
    test('Should return an empty object if the condition is false', () => {
        const mountedIf = shallow(<If condition={false}/>);
        const result = {}
        expect(mountedIf).toEqual(result);
    })
    test('Should return the children if the condition is true', () => {
      const mountedIf = shallow(<If condition={true}/>);
      const result = {};
      expect(mountedIf).toEqual(result);
  })
  });
});