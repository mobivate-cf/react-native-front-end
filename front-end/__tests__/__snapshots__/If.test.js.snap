// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`If Component Rendering Should follow snapshot 1`] = `null`;
